gradle/versions.gradle

```diff
@@ -147,6 +147,8 @@ dependencyManagement {
       entry 'tuweni-units'
     }
 
+    dependency 'commons-net:commons-net:3.9.0'
```

Comment:
Does it have parent dependency? Would it be good to add a comment to upgrade that in the future and remove this.

Reply:
done